public/app/features/canvas/runtime/scene.tsx

```diff
@@ -53,6 +54,24 @@ export class Scene {
     this.root = this.load(cfg, enableEditing);
   }
 
+  getNextElementName = (isGroup = false) => {
+    const label = isGroup ? 'Group' : 'Element';
+    let idx = this.byName.size;
+
+    while (true && idx < 100) {
```

Comment:
fixed at 100 seems odd -- it seems possible we could eventually have 100+ elements.

Maybe save the initial max, and set that as idx+100?